### ISSUE TITLE
Add `near-workspaces` logic to test/demo some of the examples

### ISSUE THREAD

**Issue by dj8yfo** [near/near-sdk-rs#1256]
Some of the examples are tested via only private interface, and not tested against a live sandbox node

- [ ] uncomment previsously `#[ignore]`-d assertions
  - [ ] examples/adder/src/lib.rs
    - https://github.com/dj8yfo/near-sdk-rs/blob/b4b3dff0ef5a3f30841348a80461efc56f97945f/examples/adder/src/lib.rs#L63-L85
- [ ] add tests/demonstration of example's logic via running in sandbox `near_workspaces` and contract's public interface
  - [ ] examples/versioned/src/lib.rs
    - https://github.com/dj8yfo/near-sdk-rs/blob/b4b3dff0ef5a3f30841348a80461efc56f97945f/examples/versioned/src/lib.rs?plain=1#L166
  - [ ] examples/mission-control/src/mission_control.rs:102
    - https://github.com/dj8yfo/near-sdk-rs/blob/b4b3dff0ef5a3f30841348a80461efc56f97945f/examples/mission-control/src/mission_control.rs?plain=1#L120
  - [ ] examples/test-contract/src/lib.rs:50
    - https://github.com/dj8yfo/near-sdk-rs/blob/b4b3dff0ef5a3f30841348a80461efc56f97945f/examples/test-contract/src/lib.rs?plain=1#L68
  - [ ] examples/status-message/src/lib.rs:76
    - https://github.com/dj8yfo/near-sdk-rs/blob/b4b3dff0ef5a3f30841348a80461efc56f97945f/examples/status-message/src/lib.rs?plain=1#L94

Each individual example can be addressed separately via a separate pr. 
This task can be picked up after #1253 is merged

**Comment by r-near** [comment]
Resolved — near-workspaces tests now exist across many examples (versioned, mission-control, test-contract, status-message, adder, callback-results, mpc-contract).
